test/integration/join-test.js

```diff
@@ -107,15 +109,22 @@ testRingpopCluster({
 });
 
 testRingpopCluster({
-    size: 25
+    size: 25,
 }, 'mega cluster', function t(bootRes, cluster, assert) {
-    assert.equal(cluster.length, 25, 'cluser of 25');
-
-    cluster.forEach(function eachNode(node) {
-        assert.ok(node.isReady, 'node is bootstrapped');
+    cluster.forEach(function eachRingpop(ringpop) {
+            ringpop.gossip.stop();
     });
 
-    assert.end();
+    // wait 50 milliseconds to let pings die out, otherwise tchannel error 
```

Comment:
Since the protocolperiod is set to 1ms, a lot of pings are flying around. After convergence the protocol period is set back to the default but it takes a moment for all the 1ms pings to be handled. If we destroy the channels at this moment, a tchannel error will be generated. This was the quick fix.